wicket-native-websocket/wicket-native-websocket-core/src/main/java/org/apache/wicket/protocol/ws/api/IWebSocketProcessor.java

```diff
@@ -16,13 +16,20 @@
  */
 package org.apache.wicket.protocol.ws.api;
 
+import org.apache.wicket.protocol.http.WebApplication;
+import org.apache.wicket.protocol.ws.WebSocketSettings;
+
 /**
  * Processes web socket messages.
  *
  * @since 6.0
  */
 public interface IWebSocketProcessor
 {
+	default void onOpen(IWebSocketSession webSocketSession, final WebApplication application) {
```

Comment:
Please add javadoc

Reply:
@martin-g 

I will. I just wanted to get your opinion on the whole thing and double check if there was another way to do this.